ui/job-view/headerbars/SecondaryNavBar.jsx

```diff
@@ -342,6 +341,9 @@ class SecondaryNavBar extends React.PureComponent {
 
             <span>
               <span
```

Comment:
Could this one also be a ``reactstrap`` ``Button`` ?

Reply:
Okay, I will change it too.

Reply:
Will it be okay to change this too,? 
If yes, will the `tabindex` attribute be deleted? I am curious since the tabindex is a negative value here.

https://github.com/mozilla/treeherder/blob/84ff28b6ae43af03bf1bc955c2bbccfd6c30d263/ui/job-view/headerbars/SecondaryNavBar.jsx#L250-L265

Reply:
This too:

https://github.com/mozilla/treeherder/blob/84ff28b6ae43af03bf1bc955c2bbccfd6c30d263/ui/job-view/headerbars/SecondaryNavBar.jsx#L293-L308

Reply:
I suggest you look what `tabIndex` is used for and see if it's needed with the react-strap `Button` component (inspecting an element will help with this).

Reply:
The react-strap `Button` component defaults to `tabIndex="0"`. I think it's needed.